src/Symfony/Component/Intl/Tests/Transliterator/EmojiTransliteratorTest.php

```diff
@@ -82,30 +84,6 @@ public static function provideTransliterateTests(): iterable
         ];
     }
 
-    /**
-     * @dataProvider provideLocaleTest
-     */
-    public function testAllTransliterator(string $locale)
-    {
-        $tr = EmojiTransliterator::create($locale);
-
-        $this->assertNotEmpty($tr->transliterate('😀'));
-    }
-
-    public static function provideLocaleTest(): iterable
```

Comment:
I intentionally kept most tests as legacy tests, to ensure that the BC layer works fine (and I added the assertion about actually instantiating the requested class as my first version of the BC layer was broken in that regard, which could break type checks for existing code using the old class as type). I only removed this test because it relied on the data to find all supported locales, and the data is gone from this component. This is still tested in the new component.